### PR TITLE
Added option to forbid flipping of rectangle to MaxRectsBinPack

### PR DESCRIPTION
**Pull request by RebelMoogle**
Found it useful to disable flipping. added an option to the init and constructor to disable flipping.

### CODE REVIEW

File: MaxRectsBinPack.cpp [hunk 1]
```diff
@@ -25,13 +25,14 @@ binHeight(0)
 {
 }
 
-MaxRectsBinPack::MaxRectsBinPack(int width, int height)
+MaxRectsBinPack::MaxRectsBinPack(int width, int height, bool allowFlip) : m_AllowFlip(allowFlip)
 {
 	Init(width, height);
```
Comment (juj) [inline MaxRectsBinPack.cpp]: This should call `Init(width, height, allowFlip);` instead, since otherwise it overrides allowFlip to true always.

**Comment by juj** [comment]
Thanks, looks ok, some comments inline.

File: MaxRectsBinPack.h [hunk 2]
```diff
@@ -22,11 +22,11 @@ class MaxRectsBinPack
 	MaxRectsBinPack();
 
 	/// Instantiates a bin of the given size.
-	MaxRectsBinPack(int width, int height);
+	MaxRectsBinPack(int width, int height, bool allowFlip = true);
```
Comment (juj) [inline MaxRectsBinPack.h]: Would be nice to have a docline "/// @param allowFlip Specifies whether the packing algorithm is allowed to rotate the input rectangles by 90 degrees to consider a better placement." or something like that?

**Comment by RebelMoogle** [comment]
Sorry for responding so late, I made the changes as per your review.

**Comment by juj** [comment]
Great, thanks!

**Pull request merged**